pkg/collector/corechecks/sbom/check.go

```diff
@@ -173,13 +180,25 @@ func (c *Check) Run() error {
 			c.processor.processContainerImagesRefresh(c.workloadmetaStore.ListImages())
 		case <-hostPeriodicRefreshTicker.C:
 			c.processor.processHostRefresh()
+		case <-metricTicker.C:
+			c.sendUsageMetrics()
 		case <-c.stopCh:
 			c.processor.stop()
 			return nil
 		}
 	}
 }
 
+func (c *Check) sendUsageMetrics() {
+	c.sender.Count("datadog.agent.sbom.container_images.running", 1.0, "", nil)
```

Comment:
This seems to be a duplicate of `datadog.agent.checks.runs`?

Reply:
Is this metric tagged with the check ? Anyway, I don't think we can use it as it's a long running check :-/